lib.php

```diff
@@ -106,27 +106,23 @@ public function use_editor($elementid, array $options = null, $fpoptions = null)
         foreach ($groups['style1'] as $plugin) {
             $groupplugins[] = array('name' => $plugin, 'params' => array());
         }
```

Comment:
We aren't using $groupplugins and $group. We can clean up some unnecessary code from line 87 to 108, though we might want to throw the exception when $options['supsub'] is invalid?